src/pages/index.js

```diff
@@ -219,13 +224,18 @@ const cardList = new Section(
 //userinfo instance
 const userInfo = new UserInfo(profileTitle, profileDescription, profilePicture);
 
-api.getInitialCards().then((cards) => {
-  cardList.renderItems(cards);
-});
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardList.renderItems(cards);
+  })
+  .catch((err) => {
+    console.error(err);
+  });
 
 api.usersInfo().then((userData) => {
   userInfo.setUserInfo(userData.name, userData.about);
-  userInfo.setAvatarInfo(userData.avatar)
+  userInfo.setAvatarInfo(userData.avatar);
 });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**